### ISSUE TITLE
Impossible Package selection at attended sysupgrade for cudy,wr2100

### ISSUE THREAD

**Issue by woklei** [openwrt/asu#569]
Reporting here as requested by the upgrade dialog:

{
    "url": "https://sysupgrade.openwrt.org",
    "revision": "r20134-5f15225c1e",
    "advanced_mode": "1",
    "branch": "22.03",
    "efi": null,
    "profile": "cudy,wr2100",
    "target": "ramips/mt7621",
    "version": "23.05.0-rc3",
    "packages": [
        "auc",
        "base-files",
        "busybox",
        "ca-bundle",
        "cgi-io",
        "collectd-mod-ping",
        "dnsmasq",
        "dropbear",
        "firewall4",
        "fstools",
        "fwtool",
        "getrandom",
        "hostapd-common",
        "iwinfo",
        "jansson",
        "jshn",
        "jsonfilter",
        "kernel",
        "kmod-crypto-crc32c",
        "kmod-crypto-hash",
        "kmod-gpio-button-hotplug",
        "kmod-leds-gpio",
        "kmod-lib-crc-ccitt",
        "kmod-lib-crc32c",
        "kmod-mt7603",
        "kmod-mt7615-firmware",
        "kmod-mt7615e",
        "kmod-nf-conntrack",
        "kmod-nf-conntrack6",
        "kmod-nf-flow",
        "kmod-nf-log",
        "kmod-nf-log6",
        "kmod-nf-nat",
        "kmod-nf-reject",
        "kmod-nf-reject6",
        "kmod-nfnetlink",
        "kmod-nft-core",
        "kmod-nft-fib",
        "kmod-nft-nat",
        "kmod-nft-offload",
        "kmod-ppp",
        "kmod-pppoe",
        "kmod-pppox",
        "kmod-slhc",
        "libblobmsg-json",
        "libc",
        "libiwinfo",
        "libiwinfo-data",
        "libiwinfo-lua",
        "libjson-c",
        "libjson-script",
        "liblua",
        "liblucihttp",
        "liblucihttp-lua",
        "libmnl",
        "libnftnl",
        "libnl-tiny",
        "libopenssl",
        "libubox",
        "libubus",
        "libubus-lua",
        "libuci",
        "libuclient",
        "libucode",
        "libustream-wolfssl",
        "libwolfssl",
        "logd",
        "lua",
        "luci",
        "luci-app-attendedsysupgrade",
        "luci-app-ddns",
        "luci-app-firewall",
        "luci-app-opkg",
        "luci-app-statistics",
        "luci-app-wireguard",
        "luci-base",
        "luci-i18n-attendedsysupgrade-de",
        "luci-lib-base",
        "luci-lib-ip",
        "luci-lib-jsonc",
        "luci-lib-nixio",
        "luci-mod-admin-full",
        "luci-mod-network",
        "luci-mod-status",
        "luci-mod-system",
        "luci-proto-ipv6",
        "luci-proto-ppp",
        "luci-ssl",
        "luci-theme-bootstrap",
        "mtd",
        "netifd",
        "nftables-json",
        "odhcp6c",
        "odhcpd-ipv6only",
        "openssh-sftp-server",
        "openwrt-keyring",
        "opkg",
        "ppp",
        "ppp-mod-pppoe",
        "procd",
        "procd-seccomp",
        "procd-ujail",
        "px5g-wolfssl",
        "qrencode",
        "rpcd",
        "rpcd-mod-file",
        "rpcd-mod-iwinfo",
        "rpcd-mod-luci",
        "rpcd-mod-rrdns",
        "ubi-utils",
        "ubox",
        "ubus",
        "ubusd",
        "uci",
        "uclient-fetch",
        "ucode",
        "ucode-mod-fs",
        "ucode-mod-ubus",
        "ucode-mod-uci",
        "uhttpd",
        "uhttpd-mod-ubus",
        "urandom-seed",
        "urngd",
        "usign",
        "wpad-basic-wolfssl",
        "zerotier"
    ],
    "diff_packages": true,
    "filesystem": "squashfs",
    "client": "luci/git-23.093.42303-58b861d"
}

STDERR:

Package list missing or not up-to-date, generating it.

Building package index...
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/targets/ramips/mt7621/packages/Packages.gz
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/openwrt_core
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/targets/ramips/mt7621/packages/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/base/Packages.gz
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/openwrt_base
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/base/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/luci/Packages.gz
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/openwrt_luci
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/luci/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/packages/Packages.gz
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/openwrt_packages
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/packages/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/routing/Packages.gz
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/openwrt_routing
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/routing/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/telephony/Packages.gz
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/openwrt_telephony
Downloading https://downloads.openwrt.org/releases/23.05.0-rc3/packages/mipsel_24kc/telephony/Packages.sig
Signature check passed.
Downloading file:packages/Packages
Updated list of available packages in /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/../../../../../../../../../../../home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/dl/imagebuilder
Downloading file:packages/Packages.sig
Signature check failed.
Remove wrong Signature file.
Collected errors:
 * check_data_file_clashes: Package libustream-mbedtls20201210 wants to install file /home/aparcar/asu/worker1/cache/23.05.0-rc3/ramips/mt7621/build_dir/target-mipsel_24kc_musl/root-ramips/lib/libustream-ssl.so
	But that file is already provided by package  * libustream-wolfssl20201210
 * opkg_install_cmd: Cannot install package luci-ssl.
make[2]: *** [Makefile:189: package_install] Error 255
make[1]: *** [Makefile:154: _call_manifest] Error 2
make: *** [Makefile:274: manifest] Error 2

**Comment by woklei** [comment]
Similar to #570 but different hardware

**Comment by woklei** [comment]
Solved following [this](https://forum.openwrt.org/t/not-able-to-upgrade-to-23-05-from-day-1/175346/3)  advice and additionally deselecting luci-ssl package in attended sysupgrade. (I could even install luci-ssl again after the upgrade)

I.e. I removed
- libustream-wolfssl
- libwolfssl
- luci-ssl
- px5g-wolfssl
- wpad-basic-wolfssl

from the package list in attended sysupgrade and added
- libustream-mbedtls
- wpad-basic-mbedtls

instead.
